iib/web/migrations/versions/04dd7532d9c5_polymorphic_requests.py

```diff
@@ -0,0 +1,382 @@
+        sa.ForeignKeyConstraint(['request_rm_id'], ['request_rm.id']),
+        sa.PrimaryKeyConstraint('request_rm_id', 'operator_id'),
+        sa.UniqueConstraint('request_rm_id', 'operator_id'),
+    )
+    with op.batch_alter_table('request_rm_operator') as batch_op:
+        batch_op.create_index(
+            batch_op.f('ix_request_rm_operator_operator_id'), ['operator_id'], unique=False,
+        )
+        batch_op.create_index(
+            batch_op.f('ix_request_rm_operator_request_rm_id'), ['request_rm_id'], unique=False,
+        )
+
+    with op.batch_alter_table('request_bundle') as batch_op:
+        batch_op.drop_index('ix_request_bundle_image_id')
+        batch_op.drop_index('ix_request_bundle_request_id')
+
+    _upgrade_data()
+
+    op.drop_table('request_bundle')
+    with op.batch_alter_table('request_operator') as batch_op:
+        batch_op.drop_index('ix_request_operator_operator_id')
+        batch_op.drop_index('ix_request_operator_request_id')
+
+    op.drop_table('request_operator')
+    with op.batch_alter_table('request') as batch_op:
+        batch_op.drop_constraint('request_binary_image_id_fkey', type_='foreignkey')
+        batch_op.drop_constraint('request_binary_image_resolved_id_fkey', type_='foreignkey')
+        batch_op.drop_constraint('request_from_index_id_fkey', type_='foreignkey')
+        batch_op.drop_constraint('request_from_index_resolved_id_fkey', type_='foreignkey')
+        batch_op.drop_constraint('request_index_image_id_fkey', type_='foreignkey')
+        batch_op.drop_column('binary_image_resolved_id')
+        batch_op.drop_column('from_index_id')
+        batch_op.drop_column('binary_image_id')
+        batch_op.drop_column('from_index_resolved_id')
+        batch_op.drop_column('organization')
+        batch_op.drop_column('index_image_id')
+
+
+def _upgrade_data():
+    connection = op.get_bind()
+
+    connection.execute(
+        request_add_bundle_table.insert().from_select(
+            ['request_add_id', 'image_id'],
+            select([request_bundle_table.c.request_id, request_bundle_table.c.image_id]),
+        )
+    )
+
+    connection.execute(
+        request_add_table.insert().from_select(
+            [
+                'id',
+                'organization',
+                'binary_image_id',
+                'binary_image_resolved_id',
+                'from_index_id',
+                'from_index_resolved_id',
+                'index_image_id',
+            ],
+            select(
+                [
+                    old_request_table.c.id,
+                    old_request_table.c.organization,
+                    old_request_table.c.binary_image_id,
+                    old_request_table.c.binary_image_resolved_id,
+                    old_request_table.c.from_index_id,
+                    old_request_table.c.from_index_resolved_id,
+                    old_request_table.c.index_image_id,
+                ],
+                old_request_table.c.type == 1,
```

Comment:
Could you either import the enum from models.py or use variables defined at the top of the file instead of magic numbers?